visualization/dashboard/src/MLIDashboard/Controls/FilterControl.tsx

```diff
@@ -19,87 +21,118 @@ export interface IFilterControlState {
 export class FilterControl extends React.PureComponent<IFilterControlProps, IFilterControlState> {
     private static readonly classNames = mergeStyleSets({
```

Comment:
In moving to style to the Figma, we should also move to having separate .styles.ts files to hold the styles, and pull colors from the theme rather than hardcoded colors